### PR TITLE
Delete Kafka consumer group on Hermes subscription deletion

### PR DESCRIPTION
**Pull request by MarcinBobinski**

### CODE REVIEW

File: hermes-management/src/main/java/pl/allegro/tech/hermes/management/infrastructure/kafka/service/KafkaConsumerGroupManager.java [hunk 1]
```diff
@@ -80,4 +86,31 @@ public void createConsumerGroup(Topic topic, Subscription subscription) {
           e);
     }
   }
+
+  @Override
+  public void deleteConsumerGroup(Topic topic, Subscription subscription) {
```
Comment (szczygiel-m) [inline hermes-management/src/main/java/pl/allegro/tech/hermes/management/infrastructure/kafka/service/KafkaConsumerGroupManager.java:91]: Is there a need to pass the whole `Topic` and `Subscription` as a parameters?

File: hermes-management/src/test/groovy/pl/allegro/tech/hermes/management/infrastructure/kafka/service/KafkaConsumerGroupManagerSpec.groovy [hunk 2]
```diff
@@ -151,6 +157,68 @@ class KafkaConsumerGroupManagerSpec extends Specification {
         kafkaContainer.dockerClient.unpauseContainerCmd(containerId).exec()
     }
 
+    def "Should delete specified consumer group and retain others"() {
```
Comment (szczygiel-m) [inline hermes-management/src/test/groovy/pl/allegro/tech/hermes/management/infrastructure/kafka/service/KafkaConsumerGroupManagerSpec.groovy:160]: Nice. Can we also add some integration test for that mechanism? I mean the one which is creating topic and subscription (with validation of creation of consumer-group), deleting it and then checking whether consumer group was deleted too. What do You think about it?

Comment (MarcinBobinski) [reply hermes-management/src/test/groovy/pl/allegro/tech/hermes/management/infrastructure/kafka/service/KafkaConsumerGroupManagerSpec.groovy:160]: sure, good idea

**Review by moscicky — DISMISSED** [review]
Thanks for the PR. Please add a short summary in the description on how consumer group removal worked before this change and why the changes was introduced.

**Comment by MarcinBobinski** [comment]
I’ll have to close this PR and redo it because I didn’t realize that to remove a consumer group, all the consumers need to be disconnected first. 
This changes how things need to be handled, so I’ll start fresh with the correct approach.

**Pull request closed without merging**